AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/SumUpGauge.cs

```diff
@@ -42,7 +42,7 @@ public MetricTelemetry GetValueAndReset()
 
             metric.Name = this.name;
             metric.Value = this.gaugesToSum.Sum((g) => { return g.GetValueAndReset().Value; });
-            metric.Context.GetInternalContext().SdkVersion = new PerformanceCollectorModule().sdkVersion;
+            metric.Context.GetInternalContext().SdkVersion = SdkVersionAzureWebApp.VersionPrefix + SdkVersionAzureWebApp.GetAssemblyVersion();
```

Comment:
I would also suggest making a helper method or base class so you don't have so many copies of the same initialization code.